components/controls/edit/KmCkEditor.vue

```diff
@@ -197,24 +197,30 @@ export default {
       function onEditorImageUploaded(eventInfo, name, value, oldValue){
         // console.log((eventInfo, name, value, oldValue))
         //TODO: check why url is not in event args
-      	// if(value.url){
-      	// 	self.onFileUpload({data:value})
-      	// }
+      	if(value.url){
+      		self.onFileUpload({file:value})
+      	}
       }
+
+      this.$emit('onEditorReady', ed);
     },
     onEditorFocus( event, editor ) {
       // console.debug( 'Editor focused.', { event, editor } );
+        this.$emit('onEditorFocus', event, editor);
```

Comment:
Your component has to declare all events it emits using `defineEmits()` in setup script or `emits:[]` in on classic component definition:

see: https://vuejs.org/guide/components/events.html#declaring-emitted-events